db_access.py

```diff
@@ -1,19 +1,22 @@
 import sqlite3
-
+import datetime, hashlib
 from flask import app, g
 
 from chirpr import app
 
-def get_all_chirps():
+def hash_ps(text):
+    return hashlib.sha224(text.encode('utf-8')).hexdigest()
+
+
+def get_all_chirps(uid):
     conn = get_db()
-    return conn.execute('''
-        SELECT c.id, c.body, c.datetime, u.handle
-        FROM chirp c, user u
-        WHERE c.user_id = u.id
-    ''').fetchall()
+    chirps = []
+    for text in conn.execute('SELECT c.id, c.body, c.datetime, u.handle FROM chirp c, user u, followers f WHERE c.user_id = u.id ORDER BY c.id'):
```

Comment:
You still have a reference to `followers` here, that needs to be removed at this time.